### DISCUSSION TITLE
How can I realize which request is from Nuclei?

### DISCUSSION THREAD

**Discussion by huongkieutiki** [projectdiscovery/nuclei#2921] [Q&A] [answered]
I want to use Nuclei to scan my products, but I want to know which request sent by nuclei.
Is there a signature of request from nuclei? Or how can I setup for nuclei to send request with my signature?
Thank you.

**Comment by forgedhallpass** [comment, marked as answer]
You can add a custom header, or modify the `User-Agent` using the `-H` or `-header` command line arguments. You can also run the requests through a proxy (e.g. [`proxify`](https://github.com/projectdiscovery/proxify) or `Burp Suite`) in case you want to go through them easily.
